iphone/cli/commands/_build.js

```diff
@@ -2265,6 +2270,15 @@
 						cli.scanHooks(path.join(module.modulePath, 'hooks'));
 					}, this);
 
+					// Exclude arm64 architecture from simulator build in XCode 12+ - TIMOB-28042
+					if (this.legacyModules.size > 0 && parseFloat(this.xcodeEnv.version) >= 12.0) {
+						if (await processArchitecture() === 'arm64') {
```

Comment:
<!--
  1 failure:  iphone/cli/comman...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/commands/_build.js;
  Line: 2275;
-->
- :no_entry_sign: iphone/cli/commands/_build.js line 2275 – 'processArchitecture' is not defined. (no-undef)